PL2/PL2-shared/src/main/java/nl/tudelft/pl2016gr2/model/GenomeMap.java

```diff
@@ -203,8 +235,7 @@ public void trimToSize() {
   /**
    * Returns a collection of the genomes currently in the map.
    * <p>
-   * The collection is a copy of the genomes in this map.
-   * It can be used directly.
+   * The collection is a copy of the genomes in this map. It can be used directly.
```

Comment:
> Please undo these changes. It is convention to put each sentence on a new line in JavaDoc. (helps with maintainability since 1 sentence == 1 line). You should be able to set up your auto formatting to honour existing (manual) line breaks.